soh/src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -5480,6 +5511,10 @@ void Fishing_UpdateOwner(Actor* thisx, PlayState* play2) {
         case 11:
             player->actor.world.pos.z = 1360.0f;
             player->actor.speedXZ = 0.0f;
+            
+            if (CVarGetInteger("gFishingRodLeave", 0)) {
+            Fishing_RodLeave(this, play);
```

Comment:
Just a minor nitpick about whitespacing here.

Reply:
Are you suggesting I remove the whitespace above the new if statement or below? I removed the above. Tried to follow the format nearby, but it's a mixed bag.

Reply:
Sorry, guess I should have been more specific. I'm talking about needing one more tab on the `Fishing_RodLeave` line.